### PR TITLE
Accept the same initialize args as Koala

### PR DESCRIPTION
**Pull request by phallguy**
Our app uses both an access token and our applicatino secret to support signed requests to the
Facebook API. This PR updates the client to accept the same arguments for initialization as the
Koala gem so we can get the signed requests.

### CODE REVIEW

File: lib/instagram_graph_api/client.rb [hunk 1]
```diff
@@ -8,8 +8,5 @@ class Client < Koala::Facebook::API
     include InstagramGraphApi::Client::Media
     include InstagramGraphApi::Client::Discovery
 
-    def initialize(media_id = nil, access_token)
-      super(access_token)
-    end
```
Comment (phallguy) [inline lib/instagram_graph_api/client.rb:13]: It didn't look like this custom initializer was ever actually used to removed it so it just defaults to the Koala initializer.

**Review by rakeshpatra — APPROVED** [review]

**Comment by rakeshpatra** [comment]
Thank you.

**Comment by rakeshpatra** [comment]
Thank you.

**Comment by phallguy** [comment]
Any reason this was closed without merging? It seemed like a reasonable change to me.

**Comment by rakeshpatra** [comment]
@phallguy, its now merged to master branch, will update the gem soon.

**Comment by phallguy** [comment]
Perfect. Thanks for the update!

**Pull request closed without merging**